playground/frontend/playground_components/lib/src/controllers/example_loaders/examples_loader.dart

```diff
@@ -53,8 +53,11 @@ class ExamplesLoader {
   /// Loads examples from [descriptor]'s immediate list.
   ///
   /// Sets empty editor for SDKs of failed examples.
-  Future<void> load(ExamplesLoadingDescriptor descriptor) async {
-    if (_descriptor == descriptor) {
+  Future<void> load(
+    ExamplesLoadingDescriptor descriptor, {
+    bool doCheckDescriptor = true,
+  }) async {
+    if (doCheckDescriptor && _descriptor == descriptor) {
```

Comment:
1. Rename this method to `loadIfNew` or something like that.
2. Extract the loading to a new `load` method.
3. Call this new method when need to load unconditionally.